http/src/main/resources/reference.conf

```diff
@@ -256,7 +256,24 @@ azure {
      enabled = true
      database-enabled = false
      # App developers - Please keep the list of non-backward compatible versions in the list below
+     # All app versions excluded up until the switch to WSM-controlled KubernetesNamespace
+     # https://github.com/DataBiosphere/leonardo/pull/3666
```

Comment:
I generated this lists by looking at the prod DB, and selecting all versions up to and including the current version in Leo. 
(So after this PR goes in, we'll need 1 more WDS/Cromwell version bump to make them upgradeable going forward.)

Reply:
I think that probably makes sense -- however, I hope that adding to the exclusion list should be a rare thing going forward.